controllers/cards.js

```diff
@@ -1,36 +1,48 @@
+const mongoose = require('mongoose');
 const Card = require('../models/card');
 
+const { ObjectId } = mongoose.Types;
+
+const NotFoundError = require('../errors/error_not_found');
+const Error500 = require('../errors/error_500');
+
 module.exports.getAllCards = (req, res) => {
   Card.find({})
     .then((card) => {
       if (card.length === 0) {
-        return res.status(404).send({ message: 'База данных cards пуста! ' });
+        throw new NotFoundError('База данных карточек пуста!');
       }
       return res.send({ data: card });
     })
-    .catch((error) => res.status(500).send({ message: error.message }));
+    .catch(new Error500('Ошибка сервера'));
 };
 
 module.exports.createCard = (req, res) => {
-  const owner = req.user._id;
+  /* const owner = req.user._id; */
   const { name, link } = req.body;
-  Card.create({ name, link, owner })
+  Card.create({ name, link })
     .then((card) => res.send({ data: card }))
-    .catch(() => res.status(500).send({ message: 'Не удается создать карточку' }));
+    .catch(new Error500({ message: 'Не удается создать карточку' }));
```

Comment:
Нужно исправить: ошибка не обрабатывается централизованным обработчиком.
Централизованная обработка ошибок запустится только если будет вызван `next`:
```suggestion
    .catch(next);
```

Только нужно не забыть объявить `next` здесь:
https://github.com/MaximMaslovVik/15_sprint/blob/2ad864e57b00f8564f3658374d392b861c3b10fb/controllers/cards.js#L20